### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by NeoCortex3** [m4dEngi/RemotePlayWhatever#173] [General]
I just wanted to say thanks.
This tool worked like a charm and let me and my friend play system shock.
I means alot to me, to be able to have some time gaming with my friends over the internet. Thanks alot!

**Comment by darkknightdetec** [comment]
Yeah I just want to echo the above post.
I got to play an old childhood PC game with my girlfriend who lives across the country tonight because of this little app.
Thank you so much!
